mir-ci/mir_ci/tests/robot/platforms/vnc/Vnc.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import base64
+import math
+import time
+from asyncio import open_connection
+from enum import IntEnum
+from io import BytesIO
+from typing import List, Optional, Tuple
+
+import asyncvnc
+from PIL import Image
+from robot.api import logger
+from robot.api.deco import keyword, library
+from RPA.Images import Images
+from RPA.recognition.templates import ImageNotFoundError
+
+
+class Button(IntEnum):
+    LEFT = 0
+    RIGHT = 1
+    MIDDLE = 2
+
+
+@library(scope="TEST")
+class Vnc:
```

Comment:
Nit: Should `Vnc` and `WaylandHid` share an interface? It seems that they're both implementing the same interface, right?

Reply:
Indeed, and some parts of `Screencopy` as well. I'll see if I can devise a common interface to reduce code duplication.

Reply:
If it's too much of a pain, feel free to skip it. This shouldn't block the PR :)